audtorch/transforms/transforms.py

```diff
@@ -487,6 +488,133 @@ def __repr__(self):
         return '{0}({1})'.format(self.__class__.__name__, options)
 
 
+class RandomMask(object):
+    r"""Randomly masks signal along axis.
+
+    The signal is masked by multiple blocks (i.e. consecutive units) size of
+    which is uniformly sampled given an upper limit on the block size.
+    The algorithm for a single block is as follows:
+
+        1. :math:`width ~ U[0, {maximum_width}]`
+        2. :math:`start ~ U[0, {signal_size} - width)`
+
+    The number of blocks is approximated by the specified `coverage` of the
+    masking and the average size of a block.
+
+    * :attr:`coverage` controls how large the proportion of masking is
+      relative to the signal size
+    * :attr:`max_width` controls the maximum size of a masked block
+    * :attr:`value` controls the value to mask the signal with
+    * :attr:`axis` controls the axis to mask the signal along
+
+    Args:
+        coverage (float): proportion of signal to mask
+        max_width (int): maximum block size. The unit depends on the signal
+            and axis. See `MaskSpectrogramTime` and `MaskSpectrogramFrequency`
+        value (float): mask value
+        axis (int): axis to mask signal along
+
+    Example:
+        >>> a = torch.empty((1, 4, 10)).uniform_(1, 2)
+        >>> t = RandomMask(0.1, max_width=1, value=0, axis=2)
+        >>> print(t)
+        RandomMask(coverage=0.1, max_width=1, value=0, axis=2)
+        >>> len((t(a) == 0).nonzero())  # number of 0 elements
+        4
+
+    """
+    def __init__(self, coverage, max_width, value, axis):
+        self.coverage = coverage
+        self.max_width = max_width
+        self.value = value
+        self.axis = axis
+
+    def __call__(self, signal):
+
+        if isinstance(signal, np.ndarray):
+            signal = torch.from_numpy(signal)
```

Comment:
That's a good point. I was wondering how we should handle it as all our other transforms are currently in `numpy`.
In general, I would prefer to not include `numpy` here, but for now it seems to be the most reasonable solution. 

The alternative would be to not include those two lines here and combine it explicitly with a `ToTensor` transform.
@phtephanx what is your experience so far and/or opinion on this?

Reply:
I'm in favor of the current solution as discussed